exercises/D-logical-operators/exercise2.js

```diff
@@ -4,8 +4,36 @@
   This program calls some functions that are either missing or incomplete.
   Update the code so that you get the expected result.
 */
+// need to put a name in the function
+// name num
+
+
+function isNegative(num) {
+  return num < 0;
+}
+
+
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10) {
+      return true;
+  } 
+}
+
+function isShortName(name) {
+  if (name.length === 6) {
+    return true;
+  } else {
+      return false;
+  }
+}
+function startsWithD(firstLetter) {
```

Comment:
This function is incorrect, unfortunately .match() won't work in this scenario. Because the argument is a string you need to think about extracting out the first character first and then asserting whether that character is a d or not with a boolean conditional. Extracting out the first character of a string is easily googleable but let me know if you struggle